src/vm/peimagelayout.cpp

```diff
@@ -3,12 +3,16 @@
 // See the LICENSE file in the project root for more information.
 
 
-// 
+//
 
 #include "common.h"
 #include "peimagelayout.h"
 #include "peimagelayout.inl"
 
+#if !defined(PLATFORM_UNIX) && !defined(CROSSGEN_COMPILE)
+#include "amsi.h"
```

Comment:
Style thing (and I am not sure of the answer) do we usual do excludes for predefines?  Or add includes.  In this case it would be Win32 and Win64 only.

Reply:
@jkotas would you be the one with the answer? Is there a standard style we prefer here?

Reply:
We do not have 100% uniform style for this. I think the most common style for this is `#if !defined(FEATURE_PAL)`, but `#if !defined(PLATFORM_UNIX)` is used as well.

I think it would be more readable as `#if defined(PLATFORM_WINDOWS) ...`, but we do not have `PLATFORM_WINDOWS` defined for Windows build. Feel free to add to https://github.com/dotnet/coreclr/blob/master/configurecompiler.cmake if you would like. It is confusing that we do not have `PLATFORM_WINDOWS` defined for Windows.

When everything else is equal, the code with no or fewer negated conditions is better for readability.

Reply:
Updating to define/use `PLATFORM_WINDOWS`